src/components/AttendantForm.jsx

```diff
@@ -0,0 +1,320 @@
+import React, { useState } from "react";
+import Attendant from "./Attendant";
+import ErrorMessages from "../constants/errorMessages";
+
+const AttendantForm = ({
+  isAttendantsLoading,
+  isJobTitlesLoading,
+  submitAttendant,
+  attendants,
+  jobTitles,
+  attendantsApiError,
+  jobTitlesApiError,
+}) => {
+  const [name, setName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [jobTitle, setJobTitle] = useState("");
+  const [age, setAge] = useState("");
```

Comment:
You could try to store all form data in a [single state object](https://stackoverflow.com/questions/64754900/reactjs-using-object-to-store-form-data-as-state-variable). That would make the code more DRY